### DISCUSSION TITLE
How to Update when a new version of Firefox-UI-Fix releases ?

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [black7375/Firefox-UI-Fix#526] [Q&A] [answered]
Currently i am using PortableApps version of firefox and i am using Firefox-UI-Fix. Now when a update comes , i have to open `WinMerge` compare the diffs and apply changes. 
I have been having the same problem with other of  github repos that i follow like - Autohotkey Scripts, MPV Scripts as they all are just code files always have to do alot of manual work. 

Is there automation for this kind of thing ?

**Comment by black7375** [comment, marked as answer]
If you use [git](https://git-scm.com/), you can simply install & update.

- Install
  1. Install git
  2. [Open powershell](https://adamtheautomator.com/windows-open-powershell-in-a-folder/) at `FIREFOX_PROFILE` directory
  3. Use `From Git` of the [installation guide](https://github.com/black7375/Firefox-UI-Fix/wiki/Installation-Guide).
- Update
  1. [Open powershell](https://adamtheautomator.com/windows-open-powershell-in-a-folder/) at `FIREFOX_PROFILE/chrome` directory
  2. Run [`git pull`](https://git-scm.com/docs/git-pull)

**Comment by KAGEYAM4** [comment]
Thank you , will learn how to use git then.
